src/client/src/containers/Details/index.tsx

```diff
@@ -154,13 +197,27 @@ const Details = ({
                             const licenseObject = license as ILicenseObject;
                             return (
                               <p key={license + idx.toString()}>
-                                <a
-                                  className={styles.link}
-                                  href={licenseObject.url}
-                                  onKeyUp={keyUpHandler}
+                                <button
+                                  className={styles.licenseButton}
+                                  onClick={() =>
+                                    openRedirectModal({
+                                      redirectLink: String(licenseObject.url),
+                                      redirectLinkLabel: intl.formatMessage(
+                                        messages.redirectLabel,
+                                        {
+                                          redirectLabel: `${
```

Comment:
Same internationalization comment here

Reply:
This complete thing has been internationalized here. Everything is inside this:
`redirectLinkLabel: intl.formatMessage(`